### PR TITLE
My portfolio Setup and mobile first

### PR DESCRIPTION
**Pull request by tsigewossne**
##In this project
####I have created the following major things:

- An index.html file that includes the content of the portfolio
- A styles.css file for styling the project as required and mobile view
- a linters configuration to check the validity of HTML and CSS
- A descriptive readme file that explains about the project

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
```
Comment (joel-kalema) [inline index.html:2]: - [ ] You did great work 😃, however, you need to fix some parts. For good syntax, Kindly make sure you follow this structure:

`````
<body>
    <main>
        <header>
               <a href="#">LOGO</a>
               <nav>
                    --- hamburger menu ---
               </nav>
        </header>
        <section>
               --- the headline section (right after the header) ----
        </section>
    </main>
</body>
`````


- [ ]  Kindly make sure your website should look exactly like the design (e.g. font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48-988&t=OmND1dx8utbvJmxn-0) as it is [required](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md#:~:text=Your%20website%20should%20look%20exactly%20like%20the%20design%20(e.g.%20font%2C%20colors%2C%20images%2C%20text%2C%20margins)%20using%20the%20templates%20in%20Figma.) 😃,

> | | Figma template | | Your project | |
> |-|---------|-|--------|-|
> | |Screenshot 2023-05-04 110819| |Screenshot 2023-05-04 1| |

File: index.html [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+        <h1>Hey there. <br>
+            I’m Tsige.</h1>
+        <h1 class="green">I’m a software developer! </h1>
+           <p>I can help you build any software as your need
+            including DesktopApp ,Mobile feature or website.
+            For more Please Look through 
+            some of my work and experience that I have!</p>
+        <h3>
+            <a href="">Let's Contact</a> 
+        </h3>
```
Comment (joel-kalema) [inline index.html]: - [ ] Kindly, make sure you remove this line, because, it is not a part of the design 😃

**Review by joel-kalema — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @tsigewossne 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Follow GitHub flow :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me @joel-kalema  when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by tsigewossne** [comment]
#####Dear @joel-kalema thanks for your comments and suggestions I have corrected my project as you guide with the following major changes:
- follow the structure and syntax for html file
- create similarity with the given Figma syntax
- remove unnecessary content from the project

File: index.html [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+            <ul>
+              <li><a href="">Home</a></li>
+              <li><a href="">Projects</a></li>
+              <li><a href="">Contribution</a></li>
+            </ul>
+          </div>
+        </div>
+      </div>
+    </div>
+  </nav>
```
Comment (usorfaitheloho) [inline index.html]: 1. [optional] Please remove unnecessary comments(line 17) from your code to have a clean code free from errors and confusion 😄 
2. As suggested by the previous reviewer, you should follow the right structure for your code . You should use the `header` for the outer section for your `toolbar`, followed by your logo in a `<a href="#"></a>` tag , then your `nav` tag with your `hamburger menu` in it . Like this:
    >  <body>
         <header>
               <a href="#">LOGO</a>
               <nav>
                    --- hamburger menu ---
               </nav>
           </header>
          <section>
               --- the headline section (right after the header) ----
           </section>
      </body>

File: index.html [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+  <section class="headline">
+    <div class="container">
+      <h1>Hey there. <br>
+        I’m Tsige.</h1>
+      <h1 class="green">I’m a software developer! </h1>
```
Comment (usorfaitheloho) [inline index.html:25]: [optional] line 38 should be in a `p` tag for SEO purposes and good semantics, you should have only one `h1` tag in your html file 😄

**Review by usorfaitheloho — CHANGES_REQUESTED** [review]
> Hi @tsigewossne  ,
> 
> Great job for implemented the some of the Project Requirement , but there are some changes that need to be implemented in order to proceed to the next project 👍
> 
> ## Highlights 👏🏻
> ✔️ No linters error ✔️ You used correct Git Hub-Flow. ✔️ Good README ✔️ Good Commit Messages
> 
> almost
> ## Required Changes ♻️
> _Check the comments below the review._
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by tsigewossne** [comment]
## Great Thanks to the Reviewers!
##### Changes I have made are:

- format the HTML file as requested by @usorfaitheloho 
- Remove unwanted comments from the html file as requested by the reviewer

File: index.html [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+                <li><a href="">Home</a></li>
+                <li><a href="">Projects</a></li>
+                <li><a href="">Contribution</a></li>
+              </ul>
+            </div>
+          </div>
+        </div>
+      </div>
+    </nav>
+  </header>
```
Comment (AbelG101) [inline index.html]: - [ ] As suggested by the previous reviewer please make sure to include a link for the logo before the `nav` tag and inside the `nav` tag you only need to place an image for the menu icon or any other way you prefer to add an icon. So please remove the code inside `<div class="menu_main">` and only place the icon ✔. 
- Here's how it should look in the end:
```
<header>
     <a href="#">LOGO</a>
     <nav>
        <img src="menu-icon">
     </nav>
</header>
```
- You can download the menu icon from Figma.

**Review by AbelG101 — CHANGES_REQUESTED** [review]
# Hi @tsigewossne 👋, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- Correct use of GitHub Flow ✔
-  Well-structured files ✔
- All linter checks are passing ✔

 ## Required Changes ♻️
- _Check the comments under the review._
- you just have to make a few changes and you're on your way 🚀.

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by tsigewossne** [comment]
### Changes that I have made now:

- Remove the nav contents and replace with header image (main menu icon)
- correcting the CSS file regarding to the HTML changes

**Review by anagudelogu — APPROVED** [review]
# STATUS: APPROVED! 🍾 🎆 🥇 

Hi @tsigewossne, 
I'm Andres.

You did a great job working on this milestone, your project is complete! 🚀 
There is nothing else to say other than... it's time to merge it 🤖  
Congratulations! 🎉 



## Highlights
✅ Great use of GitHub flow. 💯 
✅ Your project is documented professionally. 📜 
✅ Great UI design. 💻 
✅ Descriptive PR title, summary and commit messages. 💬 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- [ ] **[OPTIONAL]:** I suggest that you fix a little the UI design as currently, there's a blank space at the top of your page and I would suggest that you use the correct font-size and font-weight for your title! 🤓 🔧 

|Template|Your design|
|---|---|
| image | image |



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**